.coveragerc

```diff
@@ -232,6 +235,7 @@ omit =
     homeassistant/components/switch/mystrom.py
     homeassistant/components/switch/netio.py
     homeassistant/components/switch/orvibo.py
+    homeassistant/components/switch/pilight.py
```

Comment:
No need to add pilight here. The wildcard entry  above already takes care of that